apps/server/src/modules/server/server.config.ts

```diff
@@ -110,6 +110,7 @@ export interface ServerConfig
 	I18N__DEFAULT_LANGUAGE: LanguageType;
 	I18N__FALLBACK_LANGUAGE: LanguageType;
 	I18N__DEFAULT_TIMEZONE: Timezone;
+	IMPORTUSER_SAVE_ALL_MATCHES_REQUEST_TIMEOUT_MS: number;
```

Comment:
This should be part of the `UserImportConfig` interface in the `user-import-config.ts` in the root of the user-import module. (Needs to be created)